### PR TITLE
DOC: add the doc source file for the `yulesimon`

### LINKED ISSUE #9997: DOC: Not all stats distributions have tutorials
Not all stats distributions have tutorials.   Here's a list of missing ones.  [Since the terms `frechet_l` and `frechet_r` are deprecated, they don't need a tutorial, but the others probably do.]

```
python -c "import operator, scipy.stats; print(' '.join(['doc/source/tutorial/stats/continuous_' + dst + '.rst' for dst in sorted(set(map(operator.itemgetter(0), scipy.stats._distr_params.distcont)))]))" | xargs ls -1 2>&1 | fgrep "No such" |   awk '{print $2}'

doc/source/tutorial/stats/continuous_argus.rst:
doc/source/tutorial/stats/continuous_crystalball.rst:
doc/source/tutorial/stats/continuous_exponnorm.rst:
doc/source/tutorial/stats/continuous_frechet_l.rst:
doc/source/tutorial/stats/continuous_frechet_r.rst:
doc/source/tutorial/stats/continuous_halfgennorm.rst:
doc/source/tutorial/stats/continuous_kappa3.rst:
doc/source/tutorial/stats/continuous_kappa4.rst:
doc/source/tutorial/stats/continuous_levy_stable.rst:
doc/source/tutorial/stats/continuous_moyal.rst:
doc/source/tutorial/stats/continuous_pearson3.rst:
doc/source/tutorial/stats/continuous_skewnorm.rst:
doc/source/tutorial/stats/continuous_vonmises_line.rst:
```

### Scipy/Numpy/Python version information:
```
1.3.0.dev0+f89a187 1.16.0 sys.version_info(major=3, minor=6, micro=6, releaselevel='final', serial=0)
```

### PR DESCRIPTION
**Pull request by rlucas7**
partially closes #9997 , adds a doc source for `yulesimon`.

### CODE REVIEW

**Comment by rlucas7** [comment]
The html build works locally but my latex build does not. The latex issues (I see 12 locally) first appear in the `discrete_laplace.rst` file when I do a latex build locally. 

I'm not seeing helpful error logging in the CI build that helps me determine if the issue is the same or something else. 

It seems like this is an issue from a different commit than those on the PR here?

**Comment by rgommers** [comment]
> It seems like this is an issue from a different commit than those on the PR here?

I just tried building master and that works fine. I'll try your branch too.

**Comment by rgommers** [comment]
Hmm, `make all-pdf` worked fine on master but with your branch it indeed stops in `dlaplace`:
```
LaTeX Warning: Hyper reference `generated/scipy.stats.dlaplace:scipy.stats.dlap
lace' on page 332 undefined on input line 27119.

! Missing $ inserted.
<inserted text> 
                $
l.27126  p\left
               (k;\alpha\right) = \alpha \B\left(k, \alpha+1\right),
```
It may just be that adding `yulesimon` right below `dlaplace` triggered a difference in behavior for a missing `$` in `dlaplace`.

**Comment by ilayn** [comment]
I think it is `\B` that is undefined here..

**Comment by rlucas7** [comment]
Ilayn, 

I think you are right. I looked in beta and beta prime docs and looks like the beta function is never demoted directly. Instead always through the use of the gamma functions. I will rewrite in terms of gamma functions directly when I get home this evening.

Thanks! 

Sincerely, 

-Lucas Roberts

**Comment by rlucas7** [comment]
@ilayn and @rgommers Looks like switching `\B(a,b)` to the ratio of gamma functions removed the doc build failure.

**Comment by pvanmulbregt** [comment]
Many of the tutorials for the stats distributions follow a similar format.  See https://docs.scipy.org/doc/scipy/reference/tutorial/stats/discrete_bernoulli.html for an example.
(The consistency of the discrete tutorials is not as good as it is for the continuous ones.)
Do you think you could modify this tutorial to match that layout?

**Comment by ilayn** [comment]
Also you could have just type `B(a, b)` or `\beta(a,b)`. I just wanted to point that `\B` is not a valid macro in LaTeX. But if gamma definition is better nice to have consistency.

**Comment by rlucas7** [comment]
Yeah it looks like the normalizing constant in the binomial cumulative mass function uses the ratio of gamma functions so this should be consistent with what I see in other discrete distributions. 

Sincerely, 

-Lucas Roberts

**Comment by pvanmulbregt** [comment]
I like the reformatting. 
The build failure appears to be real, as it also failed when I tried to use it locally. Something in the first TeX eqnarray block is triggering an error.

File: doc/source/tutorial/stats/discrete_yulesimon.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+A Yule-Simon random variable with parameter :math:`\alpha>0`
+can be represented as a mixture of
+exponential random variates. To see this write :math:`W` as an exponential
+random variate with rate :math:`\rho` and a Geometric random variate :math:`K`
+with probability :math:`1-exp(-W)` then :math:`K` marginally has a Yule-Simon
+distribution. The latent variable representation described above is used for
+random variate generation.
+
+.. math::
+
```
Comment (pvanmulbregt) [inline doc/source/tutorial/stats/discrete_yulesimon.rst]: The `math::` block below has `:nowrap:`.  This one doesn't. Perhaps it needs it?

**Comment by rlucas7** [comment]
The build issue on commit ` 4dacf4b` is weird, the log states a missing `\right)` but I they all look balanced to me and the build completed locally in my dev environment just fine. 

I know when I run `python setup.py build_sphinx` it is building the html versions of the files, i'm not sure if the html build uses latex or not, presumably `sphinx` is using latex under the hood but it is difficult for me to ascertain if my assumption is true. The issues in building seem to be solely from latex.

**Comment by rlucas7** [comment]
ci/circleci: pypy3  build failure looks unrelated (numpy issue?). 

Otherwise this fixes the issues, the docs build successfully. I added a `:nowrap:` as suggested by @pvanmulbregt and fixed the unmatched `\left(` by removing a `\left)` and making it a `\right)`. 

Let me know if anything else is needed for the tutorial doc. Would be good to have this in so the distribution is known and used.

**Review by pvanmulbregt — APPROVED** [review]

**Comment by ilayn** [comment]
pypy run on the CI is removed, you can ignore it

**Comment by rlucas7** [comment]
Anything else needed to get this PR merged? 

Sincerely, 

-Lucas Roberts

**Comment by pvanmulbregt** [comment]
Thanks @rlucas7. (Nothing else needed, the main line has just opened up again after the freeze related to the 1.3 release.)

**Comment by rlucas7** [comment]
Ah, I didn’t know one can freeze a branch in github. Good to know. Thanks for merging-and the feedback. 

Sincerely, 

-Lucas Roberts

**Pull request merged**